pkg/api/install/install.go

```diff
@@ -59,15 +59,33 @@ func init() {
 
 		case *watchv1.Event:
 			switch b := objB.(type) {
+			case *watchv1.InternalEvent:
+				return true, watchv1.Convert_versioned_Event_to_versioned_InternalEvent(a, b, s)
 			case *watchapi.Event:
 				return true, watchv1.Convert_versioned_Event_to_watch_Event(a, b, s)
 			}
+		case *watchv1.InternalEvent:
+			switch b := objB.(type) {
+			case *watchv1.Event:
+				return true, watchv1.Convert_versioned_InternalEvent_to_versioned_Event(a, b, s)
+			}
 		case *watchapi.Event:
 			switch b := objB.(type) {
 			case *watchv1.Event:
 				return true, watchv1.Convert_watch_Event_to_versioned_Event(a, b, s)
 			}
 
+		case *kapi.ListOptions:
```

Comment:
Not registered yet, will remove them once we upgrade.

On Thu, Jun 23, 2016 at 9:07 AM, David Eads <notifications@github.com>
wrote:

> In pkg/api/install/install.go
> <https://github.com/openshift/origin/pull/9498#discussion_r68230375>:
>
> >  		case *watchapi.Event:
> >  			switch b := objB.(type) {
> >  			case *watchv1.Event:
> >  				return true, watchv1.Convert_watch_Event_to_versioned_Event(a, b, s)
> >  			}
> >
> > +		case *kapi.ListOptions:
>
> Why didn't we get these from upstream? They register the same types don't
> they?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9498/files/a068e55aa25db3fc149aa1d896426365a7734df5#r68230375>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p4_iwffKTXvOcF4RMMbD5MOCVIlvks5qOoUkgaJpZM4I8Zon>
> .
>